lib/java_buildpack/container/tomcat/tomcat_geode_store.rb

```diff
@@ -44,8 +44,10 @@ def release
         credentials = @application.services.find_service(FILTER, KEY_LOCATORS, KEY_USERS)['credentials']
         user = credentials[KEY_USERS].find { |u| cluster_operator?(u) }
 
-        @droplet.java_opts.add_system_property 'gemfire.security-username', user['username']
-        @droplet.java_opts.add_system_property 'gemfire.security-password', user['password']
+        File.open(@droplet.root + 'gemfire.properties', 'w') do |f|
```

Comment:
Why `gemfire.properties` instead of `gfsecurity.properties`? https://gemfire.docs.pivotal.io/910/geode/reference/topics/gemfire_properties.html

Reply:
We did think about using `gfsecurity.properties`, but I don't think we've revisited that since we got everything functioning properly.  It would probably be better to use the `gfsecurity.properties` file, though.  I'll update everything.

Reply:
Deleted the addition of this file altogether in favor of pulling the credentials directly from `VCAP_SERVICES` in our ClientAuth class.